src/python/grpcio/grpc/experimental/aio/_interceptor.py

```diff
@@ -148,11 +150,11 @@ def __del__(self):
             else:
                 return UnaryUnaryCall(
                     request, _timeout_to_deadline(client_call_details.timeout),
-                    client_call_details.credentials, self._channel,
+                    metadata, client_call_details.credentials, self._channel,
```

Comment:
I would say better `client_call_details.metadata`

Reply:
There is a pending test in the `interceptor_test.py` file waiting to be filled :)

Reply:
Good catch! I totally missed this one.
(And I found the trailing metadata cannot be modified easily.)